server/build/webpack.js

```diff
@@ -178,6 +189,7 @@ export default async function createCompiler (dir, { hotReload = false, dev = fa
       ],
       loaders
     },
+    devtool: dev ? 'inline-source-map' : false,
```

Comment:
Some other `devtool` settings is faster, but didn't properly work as far as I tested.